part-time-schedule/week-8.md

```diff
@@ -0,0 +1,37 @@
+# Week 8
+
+## Primary focus
+
+This week the primary focus will be on Project 1 Retros and preparation for Project 
+2 (Open Source Projects).
+
+### Monday
+
+We will start the week on Monday with completing Project 1 Demos and Retros. Demos are 
+for those who did not demo their project last week and those with additional features. 
+We will also have a Review Session where you will share the things you learned while 
+working on Project 1 as well as its challenges and improvements you think can be made.
+
+
+### Wednesday
+
+On Wednesday, we'll have a contributing etiquette Workshop that will prepare us on how to 
+contribute to Open Source Projects.
+
```

Comment:
We need to mention that they should start working on their design docs at this point. It's very important that those are complete and "approved" before they start the PR process.

I think they need to identify a project earlier than Friday. Maybe figure out an issue of a project and start docs on Thursday?